src/Microsoft.AspNetCore.Routing/Patterns/RoutePatternException.cs

```diff
@@ -31,8 +39,16 @@ public RoutePatternException(string pattern, string message)
             Pattern = pattern;
         }
 
+        /// <summary>
+        /// Gets the route pattern associated with this exception.
+        /// </summary>
         public string Pattern { get; }
 
+        /// <summary>
+        /// Populates a <see cref="SerializationInfo"/> with the data needed to serialize the target object.
+        /// </summary>
+        /// <param name="info">The <see cref="SerializationInfo"/> to populate with data.</param>
+        /// <param name="context">The destination (<see cref="StreamingContext" />) for this serialization.</param>
```

Comment:
Is `inheritdoc` a usable thing or no? We used it in the SignalR docs.

Reply:
inheritdoc isn't a feature our documentation system supports